0.11.2 release (#386)
packit-service/ogr: CHANGELOG.md

# 0.11.2

## New Features

- A new method to set flags on Pagure PRs was added. (@csomh)
  - It is Pagure-specific.

>   - Other git-forges does not have that and use commit flags instead.

Reviewer: ```suggestion
  - Other git-forges do not have this as they display the flags of the head commit on PRs.
```

- CommitFlag now has created/edited properties. (@TomasJani)

## Minor

- Pagure service is used for CentOS prod/stg instances by default. (@jsakalos)

> - We now forward the specific errors from the pagure API. (@TomasTomecek)

Reviewer: ```suggestion
- We now forward the specific errors from the Pagure API. (@TomasTomecek)
```

## Fixes

- Pagination of PR comments on Pagure was fixed. (@AdarLavi)

## Internals

> - Tests were remove from the zuul gating pipeline. (@lbarcziova)

Reviewer: ```suggestion
- Tests were removed from the zuul gating pipeline. (@lbarcziova)
```

> - We now use rpmautospec for generating changelogs in Fedora. (@TomasTomecek)

Reviewer: ```suggestion
- We now use [rpmautospec] for generating changelogs in Fedora. (@TomasTomecek)

[rpmautospec]: https://pagure.io/Fedora-Infra/rpmautospec
```
